README.md

```diff
@@ -114,8 +114,8 @@ increases the times to `2.640s` for ripgrep and `10.277s` for GNU grep.
   as UTF-16, latin-1, GBK, EUC-JP, Shift_JIS and more. (Some support for
   automatically detecting UTF-16 is provided. Other text encodings must be
   specifically specified with the `-E/--encoding` flag.)
-* ripgrep supports searching files compressed in a common format (gzip, xz,
-  lzma, bzip2 or lz4) with the `-z/--search-zip` flag.
+* ripgrep supports searching files compressed in a common format (brotli, bzip2,
```

Comment:
Could you please make sure this line is wrapped to 79 columns (inclusive)? It appears to be 80 here.

Reply:
Sorry about that, fixed now :-)